packages/styles/scss/patterns/sub-patterns/content-group/_content-group.scss

```diff
@@ -22,31 +20,40 @@
         margin-bottom: $carbon--layout-05;
       }
     }
-
     @include carbon--breakpoint(lg) {
       padding-top: $carbon--layout-05;
       padding-bottom: $carbon--layout-07;
-
       &__title {
         margin-bottom: $carbon--layout-03;
       }
     }
 
     &__title {
-      padding-left: $carbon--layout-01;
       padding-right: $carbon--layout-01;
       max-width: 90%;
 
       @include carbon--type-style('expressive-heading-04', true);
     }
 
     &__children {
-      padding-left: 0;
-      padding-right: 0;
+      margin-bottom: $carbon--layout-03;
+    }
+
+    &__CTA {
```

Comment:
The card should be hanging into the gutter so the text aligns.

<img src="https://user-images.githubusercontent.com/3793636/74361282-44028f00-4d8c-11ea-88e1-d84f2335bee7.png" alt="" width="300px" />

Reply:
Okay